### PR TITLE
added a check of dueOverTime and employers votes in main function for…

### PR DESCRIPTION
**Pull request by ca7alindev**
… send slack results messages

### CODE REVIEW

**Comment by ca7alindev** [comment]
@web7evgheni please check

File: packages/meal/function/fn.go [hunk 1]
```diff
@@ -59,37 +64,67 @@ func readMealResource(dynamicClient dynamic.Interface, ctx context.Context) (str
...
+		0,
+		currentTime.Location(),
+	)
+
+	fmt.Println("Votes: ", len(employers), "from ", len(realUsers))
+	fmt.Printf("current time: %s\n", currentTime.Format("2006-01-02 15:04:05 -0700 MST"))
+	fmt.Printf("dueOrderTime: %s\n", dueOrderTime.Format("2006-01-02 15:04:05 -0700 MST"))
 
-	if dueOrderTime == "over" {
+	if currentTime.After(dueOrderTime) || len(employers) == len(realUsers) {
```
Comment (web7evgheni) [inline packages/meal/function/fn.go:127]: @ca7alindev Please use TIMESTAMP as discussed.
Please don't hesitate to ask again if you not understood what is required or what is idea.

**Review by web7evgheni — CHANGES_REQUESTED** [review]

File: packages/meal/function/fn.go (hunk 1 above)
Comment (ca7alindev) [reply packages/meal/function/fn.go:127]: @web7evgheni i just want to clarify where to use timestamp format, for example in Meal object like {dueOverTime="1257894000"}?
Currently i use 24h time format for readeability reasons and escaped from need to create another cronjob for patching dueOverTime with timestamps everyday.

Comment (ca7alindev) [reply packages/meal/function/fn.go]: @web7evgheni please check latest changes

**Review by web7evgheni — APPROVED** [review]

**Pull request merged**